documentation/client_cert.md

```diff
@@ -1,112 +0,0 @@
-# Client Certification
-
-## Server
```

Comment:
@oraNod is this no longer supported?

Reply:
@pruivo sorry, that should actually be removed in https://github.com/infinispan/infinispan-operator/pull/1165 

It's just that I failed to remove Ryan's markdown file when we added client cert authn to documentation/asciidoc.